### PR TITLE
Use `ghc-9.2.8` in CI.

### PR DESCRIPTION
**Pull request by jorisdral**

### CODE REVIEW

**Review by coot — APPROVED** [review]
LGTM

File: .github/workflows/haskell.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.2.7", "9.4.4", "9.6.1"]
+        ghc: ["8.10.7", "9.2.8", "9.4.4", "9.6.1"]
```
Comment (coot) [inline .github/workflows/haskell.yml:16]:
```suggestion
        ghc: ["8.10.7", "9.2.8", "9.4.4", "9.6.2"]
```

**Comment by jorisdral** [comment]
@coot building `9.2.7` is currently required for merging, so maybe that should be updated to `9.2.8` in the `main` branch settings

**Review by coot — APPROVED** [review]

**Comment by coot** [comment]
It seems the `9.6.2` jobs needs to finish before I can change the branch rules.

**Pull request merged**